models/xero__general_ledger.sql

```diff
@@ -13,17 +51,20 @@ with journals as (
         j.account_name,
         j.account_type,
         jt.option,
+        jt.tracking_category_id,
         j.description,
         j.gross_amount,
         j.journal_id,
         j.net_amount,
         j.tax_amount,
         j.tax_name,
-        j.tax_type
+        j.tax_type,
+        {{case_when_tracking_category}}
```

Comment:
Not a big fan of this name, but can't think of a better one easily 🙃 . Is there any good standard here?

Reply:
case_when_for_every_tracking_category?

Reply:
`tracking_category_generator`

Reply:
Discussed offline, we are going to refactor it anyway with the new GL